packages/main/src/themes/Slider.css

```diff
@@ -0,0 +1,168 @@
+.ui5-slider {
+	overflow: hidden;
+	height: 100%;
+	vertical-align: middle;
+	text-align: center;
+	box-sizing: border-box;
+	font-family: var(--sapFontFamily);
+	justify-content: space-between;
+    flex-direction: column;
+    display: inline-flex;
+}
+.ui5-slider .ui5-slider-label {
+	box-sizing: border-box;
+    height: 2rem;
+    line-height: 2rem;
+    width: var(--_ui5_slider_item_width);
+    font-size: var(--_ui5_slider_label_text_size);
+    color: var(--_ui5_slider_label_text_color);
+    text-align: center;
+    vertical-align: middle;
+}
+
+.ui5-slider .ui5-slider-arrow {
+	visibility: hidden;
+	box-sizing: border-box;
+	height: 1.5rem;
+	-webkit-user-select: none;
```

Comment:
I think nowadays user-select: none; is enough

Reply:
Done